lms/envs/common.py

```diff
@@ -1795,7 +1795,7 @@
     'certificates',
     'dashboard',
     'instructor',
-    'instructor_task',
+    'lms.djangoapps.instructor_task',
```

Comment:
bcz django migration app  does not load migrations if its app does have full relative path.